### PR TITLE
Reorder DF

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgReorderSheet.Designer.vb [hunk 1]
```diff
@@ -22,22 +22,22 @@ Partial Class dlgReorderSheet
...
-        Me.lblSheetsToReoder.Location = New System.Drawing.Point(12, 21)
-        Me.lblSheetsToReoder.Name = "lblSheetsToReoder"
-        Me.lblSheetsToReoder.Size = New System.Drawing.Size(100, 23)
-        Me.lblSheetsToReoder.TabIndex = 0
-        Me.lblSheetsToReoder.Text = "Sheets to Reoder"
+        Me.lblDataFrameToReorder.Location = New System.Drawing.Point(10, 22)
+        Me.lblDataFrameToReorder.Name = "lblDataFrameToReorder"
+        Me.lblDataFrameToReorder.Size = New System.Drawing.Size(132, 12)
+        Me.lblDataFrameToReorder.TabIndex = 0
+        Me.lblDataFrameToReorder.Text = "Data Frames to Reorder:"
```
Comment (lilyclements) [inline instat/dlgReorderSheet.Designer.vb:36]: Added colon, corrected spelling mistake and changed Sheets to Data Frames

File: instat/dlgReorderSheet.vb [hunk 2]
```diff
@@ -22,7 +22,7 @@ Public Class dlgReorderSheet
     Private Sub dlgReorderSheet_Load(sender As Object, e As EventArgs) Handles MyBase.Load
         autoTranslate(Me)
         If bFirstLoad Then
-            initialiseDialog()
+            InitialiseDialog()
```
Comment (lilyclements) [inline instat/dlgReorderSheet.vb:25]: Capitilised "I"

File: instat/dlgReorderSheet.vb [hunk 3]
```diff
@@ -31,22 +31,25 @@ Public Class dlgReorderSheet
         SetRCodeForControls(bReset)
         bReset = False
         TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
```
Comment (lilyclements) [inline instat/dlgReorderSheet.vb:36]: Moved Initialise to be the first dialog

File: instat/dlgReorderSheet.vb [hunk 4]
```diff
@@ -31,22 +31,25 @@ Public Class dlgReorderSheet
...
-    End Sub
-
-    Private Sub initialiseDialog()
-        ucrBase.iHelpTopicID = 62
-        ucrSheetsToReorder.setDataType("data frame")
-        ucrSheetsToReorder.SetParameter(New RParameter("data_frames_order", 0))
     End Sub
 
-    Public Sub SetRCodeForControls(bReset As Boolean)
+    Private Sub SetRCodeForControls(bReset As Boolean)
```
Comment (lilyclements) [inline instat/dlgReorderSheet.vb:52]: Changed Public to Private as this dialog doesn't have a sub

**Comment by lilyclements** [comment]
@stevenndungu this is ready to merge

**Comment by stevenndungu** [comment]
merged

**Pull request merged**